docs/DeveloperGuide.md

```diff
@@ -1,38 +1,462 @@
+___
+## Value proposition
+Aids property agent in tracking information related to their property which includes:
+- Filtering appropriate tenants (Based on tags e.g.: gender, occupations and age)
+- Show expenses from the rented unit
+- Monitor payment dates 
 
-## Design & implementation
+Some of the constraint includes:
+- Single owner unit (Shared ownership will be registered under one owner's name)
+- Unable to calculate tax payment
+___
+## User Stories
+| Version | As a ...         | I want to ...                         | So that I can ...                                                    |
+|---------|------------------|---------------------------------------|----------------------------------------------------------------------|
+| v1.0    | Property Manager | add properties                        | keep track of properties                                             |
+| v1.0    | Property Manager | add clients                           | keep track of clients                                                |
+| v1.0    | Property Manager | delete properties                     | prevent properties I am no longer tracking from cluttering my data   |
+| v1.0    | Property Manager | delete clients                        | prevent clients I am no longer tracking from cluttering my data      |
+| v1.0    | Property Manager | view a list of properties             | find out what and how many properties I manage                       |
+| v1.0    | Property Manager | view a list of clients                | find out what and how many clients I manage                          |
+| v1.0    | Property Manager | check the details of a property       | view the property's information                                      |
+| v1.0    | Property Manager | pair a client to a property           | record down which client is renting which property                   |
+| v1.0    | Property Manager | unpair a client to a property         | update my rental records when a client is no longer renting property |
+| v1.0    | Property Manager | save my data                          | used the data created from a previous use of the app                 |
+| v1.0    | Property Manager | quit the app                          | -                                                                    |
+| v2.0    | Property Manager | check the details of a client         | view the client's information                                        |
+| v2.0    | Property Manager | search clients using their details    | easily find specific clients                                         |
+| v2.0    | Property Manager | search properties using their details | easily find specific properties                                      |
 
+___
+## Design
 {Describe the design and implementation of the product. Use UML diagrams and short code snippets where applicable.}
+### Architecture
```

Comment:
is the main program still called Duke
<img width="735" alt="Screenshot 2022-10-27 at 5 24 37 PM" src="https://user-images.githubusercontent.com/30253872/198246846-733f8be6-e7f5-4a2f-b6b2-bf95bd6d9b32.png">